tests/components/tailwind/test_cover.py

```diff
@@ -74,3 +81,90 @@ async def test_cover_operations(
     mock_tailwind.operate.assert_called_with(
         door=ANY, operation=TailwindDoorOperationCommand.CLOSE
     )
+
+    # Test door disabled error handling
+    mock_tailwind.operate.side_effect = TailwindDoorDisabledError("Door disabled")
```

Comment:
Maybe extract this into a separate test with parametrize?

Reply:
That is more expensive, as it constantly rebuilds the HA object. 
Maybe in a later iteration / cleanup.